HdfsAccessor.go

```diff
@@ -124,7 +124,7 @@ func (this *hdfsAccessorImpl) OpenRead(path string) (ReadSeekCloser, error) {
 	}
 	reader, err2 := client.Open(path)
 	if err2 != nil {
-		//TODO: close connection
+		client.Close()
```

Comment:
Do we need to close client connection in `OpenRead` if error happens (for example, the error could be 
"file not found", that the file path is wrong.)? I understand, the connection can be re-established if needed, but do not think we should close the connection here.